app/routes/publication-api.js

```diff
@@ -88,4 +88,29 @@ router.post('/entitlementsplans', express.json(), async (req, res) => {
   return res.json(response);
 })
 
+router.post('/readers/:user_id', express.json(), async (req, res) => {
+  const {accessToken} = req.body;
+  const {user_id} = req.params;
```

Comment:
```suggestion
  const {reader_id} = req.params;
```
Move to `reader_id` across this file?

Reply:
Yeah this is a good call. I was trying to be consistent with elsewhere in the file, but it's a good opportunity to revisit the semantics throughout.